README.md

```diff
@@ -337,6 +341,18 @@ Similarly, `etcd` is version 3.4.3, latest release.
     ...
 ```
 
+## Using other RPM repositories
+
+It is possible to build your images and packages using pre-built RPMs from other repositories than the default CBL-Mariner ones. In order to inform the toolkit to access them during the build, you have to make use of the [REPO_LIST](https://github.com/microsoft/CBL-Mariner/blob/1.0/toolkit/docs/building/building.md#repo_list) argument where you specify .repo files pointing to the additional repositories.
+
+Example:
+
+```bash
+sudo make build-packages REPO_LIST="path/to/first.repo path/to/second.repo" [your_other_args]

```

Comment:
I would prefer a more full-blown example here.  A tutorial step that builds on something a little earlier, and sequentially appears immediately before "## Add Specific Pre-Built Package Version"  at original line 284 and builds on the package info at original line 250.

Use an explicit example that builds on the toolkit.  For example, add something from the CoreUI repo like LibX11.

The tutorial needs to prevent the theory and then drive a concrete example.

Reply:
I see, no problem. As we discussed offline I'll update that part once the toolkit with the repos is officially out.

Reply:
OK, let me know how this looks now.